app/javascript/helpers/countries-helpers.js

```diff
@@ -0,0 +1,113 @@
+export const UNDEFINED_ID = '__UNDEFINED__'
```

Comment:
I believe this is the only thing currently used from this file.
I would then just move it to the component where this is used and remove this file completely.